### ISSUE TITLE
Enforce 100% code coverage in all new code [Blocked: #1726, #1725, #1497]

### ISSUE THREAD

**Issue by BenHenning** [oppia/oppia-android#1727]
As a follow-up to #1726, once the codebase reaches 90% code coverage we should introduce a check requiring that all new code have 100% code coverage in an effort to move the codebase as a whole toward 100% code coverage.

Incidentally, this is also blocked on #1725 and #1497.

Note that this is considered part of shipping the app, but as a nice-to-have and not a hard blocker. Reaching 90% code coverage is the key impact that's important for the launch, and while we can live without this issue, it will help keep code coverage up leading into the launch itself.
